registry/domain/services/organization_service.py

```diff
@@ -27,19 +27,29 @@ def add_organization_draft(self, payload, username):
         return organization.to_dict()
 
     def submit_org_for_approval(self, org_uuid, username):
-        self.org_repo.change_org_status(org_uuid, OrganizationStatus.DRAFT.value, OrganizationStatus.APPROVAL_PENDING.value, username)
+        self.org_repo.change_org_status(org_uuid, OrganizationStatus.DRAFT.value,
+                                        OrganizationStatus.APPROVAL_PENDING.value, username)
         return "OK"
 
-    def publish_org(self, org_uuid, username):
+    def publish_org_ipfs(self, org_uuid, username):
         orgs = self.org_repo.get_approved_org(org_uuid)
         if len(orgs) == 0:
             raise Exception(f"Organization not found with uuid {org_uuid}")
```

Comment:
can we have something like Orgnxiation  not found exception , so that if we want we can supress the alerts if needed